ynput/ayon-core
# Refactor OCIO config handling, introduce fallback mechanism

## Changelog Description
Improved color management distribution now includes fallback options. If no published OCIO config is found, the system will default to any predefined OCIO config path.

## Additional info
- Added settings conversion for transferring older version schema.
- Added function to extract config path from profile data
- Updated global config retrieval to use new function
- Introduced fallback mechanism for missing product entities

## Testing notes:
1.This is very difficult to test. Approach me if you can so I can help you and discuss with you the way it should work.
2. For now, just deploy to your server the settings
3. Reboot server 
4. Notice that **ayon+settings://core/imageio/ocio_config_profiles/0/type** `Published product` is having additional settings for fallback. 
5. Since you do not have any published product you can just type anything into the field `Product name` it does not matter what. 
6. In fallback section set it to build in ACES1.2
7. Open Nuke or any other DCC and see that its OCIO path is pointing to the build in path. You can observer this in AYON tray console just before the DCC is opened. Look for line with `>>> [  Setting OCIO environment to config path: <path>`
